src/components/Search/SearchAutocompleteInput.tsx

```diff
@@ -231,27 +232,50 @@ function SearchAutocompleteInput(
                         onBlur={() => {
                             autocompleteListRef?.current?.updateExternalTextInputFocus(false);
                             focusedSharedValue.set(false);
+
                             onBlur?.();
                         }}
-                        isLoading={isSearchingForReports}
-                        ref={ref}
+                        ref={(ref: BaseTextInputRef) => {
+                            if (typeof forwardedRef === 'function') {
+                                forwardedRef(ref);
+                            } else if (forwardedRef && 'current' in forwardedRef) {
+                                // eslint-disable-next-line no-param-reassign
+                                forwardedRef.current = ref;
+                            }
+                            textInput.current = ref;
+                        }}
                         type="markdown"
                         multiline={false}
                         parser={parser}
                         selection={selection}
                     />
                 </View>
-                {!!value && (
-                    <Animated.View
-                        style={styles.pr3}
-                        layout={SearchFiltersButtonTransition}
-                    >
+                <Animated.View
+                    style={styles.pr3}
+                    layout={SearchFiltersButtonTransition}
+                >
+                    {value && !isSearchingForReports ? (
                         <TextInputClearButton
                             onPressButton={clearFilters}
                             style={styles.mt0}
                         />
-                    </Animated.View>
-                )}
+                    ) : (
+                        <PressableWithoutFeedback
+                            accessibilityRole={CONST.ROLE.BUTTON}
```

Comment:
```suggestion
                            role={CONST.ROLE.PRESENTATION}
```
Shall we do the same how it is for TextInput? Also let's update the `accessibilityLabel`